src/test/java/fibonacciSequenceTests/FibonacciSequenceTest.java

```diff
@@ -0,0 +1,35 @@
+package fibonacciSequenceTests;
+
+import com.tngtech.java.junit.dataprovider.*;
+import fibonacciSequence.FibonacciSequenceDefinition;
+import org.hamcrest.Matchers;
+import org.junit.*;
+import org.junit.runner.*;
+
+import static org.hamcrest.MatcherAssert.*;
+import static org.hamcrest.Matchers.*;
+
+@RunWith(DataProviderRunner.class)
+public class FibonacciSequenceTest {
+    FibonacciSequenceDefinition definition = new FibonacciSequenceDefinition();
+
+    @DataProvider
+    public static Object[][] data() {
+        return new Object[][]{
+                {10, new Integer[]{1, 1, 2, 3, 5, 8, 13, 21, 34, 55}},
+                {1, new Integer[]{1}},
+                {2, new Integer[]{1, 1}},
+        };
+    }
+
+    @Test
+    @UseDataProvider("data")
+    public void sequenceOfTen(int number, Integer[] list) {
+        assertThat(definition.defineFibonacciSequence(number), Matchers.contains(list));
```

Comment:
dataprovider items and input parameters could be simplified to getting the list size

Reply:
Changed in b7f7b7eb3c7316a5b476c68f07f63ae64ca984d3.
Redundant input 'int number' parameter was removed from method signature, and changed to list.length in assertion.